common/src/main/java/org/apache/rocketmq/common/protocol/RequestCode.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.rocketmq.common.protocol;
 
-public class RequestCode {
+import org.apache.rocketmq.remoting.protocol.RemotingSysRequestCode;
+
```

Comment:
I am not sure it's a good idea that `RequestCode` in `common` module has a dependency `RemotingSysRequestCode` in `remoting` module.